### PR TITLE
button: update design

### PR DESCRIPTION
**Pull request by sipayRT**

### CODE REVIEW

**Comment by sipayRT** [comment]
gemini tests will be fixed after review

File: common.blocks/button/button.tests/simple.bemjson.js [hunk 1]
```diff
@@ -397,6 +397,18 @@
             ' ',
             {
                 block : 'button',
+                mods : { theme : 'islands', size : 'm', togglable : 'radio', disabled : true },
+                text : ' radio disabled'
```
Comment (tadatuta) [inline common.blocks/button/button.tests/simple.bemjson.js:401]: extra space at the beginning of the string

**Comment by tadatuta** [comment]
lgtm, gemini tests and rebase needed

File: design/common.blocks/button/_theme/button_theme_islands.styl [hunk 2]
```diff
@@ -142,21 +142,35 @@ ctx = '.button_theme_islands'
...
 
-        &.button_hovered
+        &.button_focused
         {
-            background: rgba(159, 127, 0, 0.8);
+            box-shadow: none;
+
+            &:before
+            {
+                animation: button-action-focus 0.5s infinite linear alternate;
```
Comment (GreLI) [inline design/common.blocks/button/_theme/button_theme_islands.styl:172]: It worth adding `background-color` for browsers that don't support animations (IE9). `animation` has top priority, thus it'll not affect supporting browsers.

**Comment by tadatuta** [comment]
Close in favor of https://github.com/bem/bem-components/pull/1771

**Pull request closed without merging**